### DISCUSSION TITLE
Barcode / Qrcode scan support in CameraView

### DISCUSSION THREAD

**Discussion by shugaoye** [CommunityToolkit/Maui#2160] [New Feature Discussions]
Can we add Barcode / QrCode scan to [CameraView](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/camera-view?tabs=windows)? We can refer to the implementation of [Camera.MAUI](https://github.com/hjam40/Camera.MAUI).

**Comment by acaliaro** [comment]
there are already some discussions about this 

#1687 #1998
